socorro/external/postgresql/models.py

```diff
@@ -14,11 +14,23 @@
 from sqlalchemy.orm import relationship
 import sqlalchemy.types as types
 
-try:
-    from sqlalchemy.dialects.postgresql import *
-    from sqlalchemy.dialects.postgresql.base import ischema_names
-except ImportError:
-    from sqlalchemy.databases.postgres import *
+from sqlalchemy.dialects.postgresql import (
+    INTERVAL,
+    INTEGER,
+    TEXT,
+    TIMESTAMP,
+    NUMERIC,
+    REAL,
+    DATE,
+    BOOLEAN,
+    UUID,
+    VARCHAR,
+    ARRAY,
+    BIGINT,
+    SMALLINT,
+    CHAR,
+)
+from sqlalchemy.dialects.postgresql.base import ischema_names
```

Comment:
What do you mean by "global solution to flake8"? Are you asking where we're putting all the flake8 configuration?